apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/charSequenceAssertions.kt

```diff
@@ -96,6 +96,32 @@ fun <T : CharSequence> Expect<T>.containsNot(expected: Any, vararg otherExpected
 fun <T : CharSequence> Expect<T>.containsRegex(pattern: String, vararg otherPatterns: String): Expect<T> =
     contains.atLeast(1).regex(pattern, *otherPatterns)
 
+/**
+ * Expects that the subject of the assertion (a [CharSequence]) contains a sequence which matches the given
+ * regular expression [pattern] as well as the [otherPatterns] (if given), using a non disjoint search.
+ *
+ * It is a shortcut for `contains.atLeast(1).regex(pattern, *otherPatterns)`.
+ *
+ * By non disjoint is meant that `'aa'` in `'aaaa'` is found three times and not only two times.
+ * Also notice, that it does not search for unique matches. Meaning, if the input of the search is `'ab'` and [pattern]
+ * is defined as `'a(b)?'` and one of the [otherPatterns] is defined as `'a(b)?'` as well, then both match, even though
+ * they match the same sequence in the input of the search. Use an option such as [atLeast], [atMost] and [exactly] to
+ * control the number of occurrences you expect.
+ *
+ * Meaning you might want to use:
+ *   `contains.exactly(2).regex('a(b)?')`
+ * instead of:
+ *   `contains.atLeast(1).regex('a(b)?', 'a(b)?')`
+ *
+ * @param pattern The pattern which is expected to have a match against the input of the search.
+ * @param otherPatterns Additional patterns which are expected to have a match against the input of the search.
+ *
+ * @return This assertion container to support a fluent API.
+ * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ */
+fun <T : CharSequence> Expect<T>.containsRegex(pattern: Regex, vararg otherPatterns: Regex): Expect<T> =
+    containsRegex(pattern.toString(), *otherPatterns.map { it.toString() }.toTypedArray())
```

Comment:
as mentioned in KDoc, it is a shortcut for `contains.atLeast(1).regex(pattern, *otherPatterns)` please implement it this way (same as in line 97)

Reply:
Ah, ok. I had originally thought of delegating to the existing one as an overload of sorts. I'll update it.